pkg/server/container_execsync.go

```diff
@@ -115,14 +116,17 @@ func (c *criContainerdService) ExecSync(ctx context.Context, r *runtime.ExecSync
 			TypeUrl: runtimespec.Version,
 			Value:   rawSpec,
 		},
+		// ExecID is not used now. Containerd client will use it.
```

Comment:
might be needed in debug scenarios... might be a good idea to log this id somewhere